web/react/components/search_bar.jsx

```diff
@@ -69,11 +72,18 @@ export default class SearchBar extends React.Component {
             results: null
         });
     }
+    handleKeyDown(e) {
+        if (this.refs.autocomplete) {
+            this.refs.autocomplete.handleKeyDown(e);
```

Comment:
The appropriate way to send data to components is to change state here and pass props to re-render.